src/ViewHelpers/RecipeViewHelper.php

```diff
@@ -4,7 +4,7 @@
 
 use App\Entities\IngredientEntity;
 
-class RecipeViewhelper
```

Comment:
There doesn’t seem to be any change here?

Reply:
The capitalisation has changed as it was not working for me